lib/rucio/common/didtype.py

```diff
@@ -126,15 +128,19 @@ def _did_from_str(self, did: str) -> None:
         Construct the DID from a string.
         :param did: string containing the DID information
         """
-        did_parts = did.split(DID.SCOPE_SEPARATOR, 1)
-        if len(did_parts) == 1:
-            self.name = did
-            self._update_implicit_scope()
-            if not self.has_scope():
-                raise DIDError('Object construction from non-splitable string is ambigious')
-        else:
-            self.scope = did_parts[0]
-            self.name = did_parts[1]
+        try:
+            self.scope, self.name = extract_scope(did)
+        except (ImportError, InvalidAlgorithmName, ConfigNotFound, NoSectionError) as e:  # Only use when the policy can not be found
+            logging.debug(f"Failure using exact_scope policy for '{did}': {type(e).__name__} - Using fallback.")
+            did_parts = did.split(DID.SCOPE_SEPARATOR, 1)
+            if len(did_parts) == 1:
+                self.name = did
+                self._update_implicit_scope()
+                if not self.has_scope():
+                    raise DIDError('Object construction from non-splitable string is ambigious')
```

Comment:
These typos were already there before the PR, but worth fixing here
```suggestion
                    raise DIDError('Object construction from non-splittable string is ambiguous')
```

Although I think it's still a bit cryptic. Maybe it could be changed to something like "Could not parse scope from string"?

Reply:
I don't love it either. May I instead suggest "Could not parse scope from did string {str} - fallback policy expects only one ':'"

Reply:
Sounds good, maybe instead of hardcoding `:` we can point to `DID.SCOPE_SEPARATOR` instead